doc/slack.md

```diff
@@ -0,0 +1,59 @@
+# Slack Integration
+
+While the SkyPortal frontend has several real-time interactions, such as with conversations using Comments and browser notifications, users may wish to receive some notifications in Slack.
+
+A Slack integration will allow SkyPortal notifications for individual users to be mirrored to a certain Slack channel for that individual. The basic idea is that someone with admin privileges in a Slack workspace will add a webhook for each SkyPortal user to a certain Slack channel. Each SkyPortal user then adds that webhook URL to their **Profile** page.
+
+## Slack App Creation: One-time Admin Set Up
+
+We first need to create a "Slack App" that acts as a conduit from SkyPortal to Slack. A logged in Slack Admin for a workspace should navigate to:
+
+[https://api.slack.com/apps?new_app=1](https://api.slack.com/apps?new_app=1)
+
+Then click on the "From Scratch" section to start the App building process:
+
+![Slack Integration Start](images/slack_integration_5.png)
+
+The Slack admin then creates a name for the application (like `SP_integration`) and creates the app in the desired workspace:
+
+![Slack Integration Create](images/slack_integration_6.png)
+
+After the app is created, click on `Incoming Webhooks` in the left-side menu and then activate webhooks with the toggle on the top right:
+
+![Slack Integration Webhook Enable](images/slack_integration_7.png)
+
+
+## Connecting SkyPortal Users to Slack
+
+### Slack Admin Steps
+
+For each user in Slack who wishes to connect to their SkyPortal notifications, the Slack admin will return to the `Incoming Webhooks` portion of the Slack app, then clicking on `Add New Webhook to Workspace`:
+
+![Slack Integration Webhook Generate](images/slack_integration_8.png)
+
+In the popup dialog choose the Slack channel for that user and copy the new Webhook URL:
```

Comment:
I think any direct message or (group) channel can be the endpoint. But since this is meant to be 1 to 1 between a Slack user and a SP user, the admin should usually only add a connection to a direct message channel.

Reply:
Fixed.